### DISCUSSION TITLE
How to separate publications into different folders?

### DISCUSSION THREAD

**Discussion by msimchowitz** [academicpages/academicpages.github.io#2332] [Q&A]
Hello! I'd like to keep publications in different folders, say by year (in, for example, _publications/recent or _publications/old). To figure this out, I've been trying to understand where the template forms "site.publications" from "_/publications", so that I can modify the code to form "site.publications.recent" and iterate over those.  Do you know where I would find the code that forms the list of publications from the filenames so that I can modify it?

Thanks!
Max

**Comment by rjzupkoii** [comment]
Do you mean something like this?

<pre>
_publications/
  manuscripts/
    one.md
    two.md
    three.md
  books/
    one.md
  posters/
    one.md
</pre>

If so you should be able to organize the Markdown files as in the example and they will still be processed and rendered to the "Publications" page.
